spotify_confidence/analysis/frequentist/confidence_computers/generic_computer.py

```diff
@@ -411,6 +411,7 @@ def _compute_differences(
         if type(level_as_reference) is not bool:
             raise ValueError(f"level_is_reference must be either True or False, but is {level_as_reference}.")
         groupby = listify(groupby)
+        levels = [tuple([tuple(l) for l in levels[0]])] # ensure contents are hashable
```

Comment:
This brakes a lot of tests unfortunately. The reason is that if you pass in strings as in `test.difference("control", "test")`, then `levels` will initially be `[('control', 'test')]` which is then converted, by this line, to `[(('c', 'o', 'n', 't', 'r', 'o', 'l'), ('t', 'e', 's', 't'))]`. So we would need a way to get around that in a robust way.

Reply:
Would a special case work? (see new commit)